pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -303,6 +307,7 @@ func TestToKubeContainerStatusWithResources(t *testing.T) {
 			},
 		},
 		"container reporting memory only": {
+			os: "linux",
```

Comment:
we could get rid off this if we skip the test only when runtime.GOOS == "windows"